src/test/javascript/spec/component/assessment-dashboard/exercise-assessment-dashboard.component.spec.ts

```diff
@@ -277,12 +279,12 @@ describe('ExerciseAssessmentDashboardComponent', () => {
     });
 
     afterEach(() => {
-        jest.clearAllMocks();
+        jest.restoreAllMocks();
     });
 
     it('should initialize', () => {
         fixture.detectChanges();
-        expect(comp).toBeDefined();
+        expect(comp).not.toBe(null);
```

Comment:
Be careful here, as this assertion is not handling the case of the component to be undefined (which would be unintended as well I think) . I'm not sure how meaningful this test is, I'd like to have @julian-christl s opinion on that.

Reply:
Yes, you can remove such tests. You may test the start configuration but the component is always defined.